CITING.md

```diff
@@ -33,17 +34,16 @@ BibTeX entry:
 
 If you use AutoGluon's multimodal functionality in a scientific publication, please cite the following paper:
 
-Shi, Xingjian, et al. ["Benchmarking Multimodal AutoML for Tabular Data with Text Fields."](https://datasets-benchmarks-proceedings.neurips.cc/paper/2021/file/9bf31c7ff062936a96d3c8bd1f8f2ff3-Paper-round2.pdf) Advances in Neural Information Processing Systems 35, Datasets and Benchmarks Track (2021).
```

Comment:
Shall we keep this and have both papers?

Reply:
Updated to keep both.